.circleci/config.yml

```diff
@@ -26,22 +28,35 @@ jobs:
           name: E2E Test
           command: ./end-end-tests/api-standards/test-bulk.bash
 
-  docker-main-release:
+  docker-release:
     <<: *defaults
     steps:
       - checkout
       - setup_remote_docker:
           version: '20.10.7'
       - run:
           name: Build Docker image
-          command: IMAGE=ghcr.io/snyk/sweater-comb TAG=optic-main ./scripts/build-docker.bash
+          environment:
+            GIT_BRANCH: << pipeline.git.branch >>
+            GIT_TAG: << pipeline.git.tag >>
+          command: |-
+            export TAG=${GIT_TAG:-${GIT_BRANCH}}
+            [ -n "$IMAGE" ] || exit 1
+            [ -n "$TAG" ] || exit 1
+            ./scripts/build-docker.bash'
       - run:
           name: Push Docker image
+          environment:
+            GIT_BRANCH: << pipeline.git.branch >>
+            GIT_TAG: << pipeline.git.tag >>
           command: |-
+            [ -n "$IMAGE" ] || exit 1
+            export TAG=${GIT_TAG:-${GIT_BRANCH}}
```

Comment:
If `GIT_TAG` is not set, I don't think this works. Did some quick bash shelling locally. may just want to use git tag?

Reply:
I _think_ it'll be OK. The ${_var_:-_default_} should evaluate to _default_ if _var_ is unset.

```
sh-3.2$ a=foo
sh-3.2$ b=bar
sh-3.2$ echo ${a:-${b:-nope}}
foo
sh-3.2$ echo ${c:-${d:-nope}}  # $c and $d unset
nope
sh-3.2$ echo ${c:-${b:-nope}}
bar
```